searchguard/rolesmapping.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import searchguard.settings as settings
+from searchguard.exceptions import *
+from searchguard.roles import check_role_exists
+
+
+def __process_data(role, properties):
+    """Private function to process API calls for the rolemapping module"""
+    create_sg_rolemapping = requests.put('{}/rolesmapping/{}'.format(settings.SEARCHGUARD_API_URL, role),
+                                         data=json.dumps(properties),
+                                         headers=settings.HEADER,
+                                         auth=settings.SEARCHGUARD_API_AUTH)
+
+    if create_sg_rolemapping.status_code == 201 or create_sg_rolemapping.status_code == 200:
```

Comment:
Could probably use the `.ok()` [method](http://docs.python-requests.org/en/master/api/#requests.Response.ok).  

Not tested, but probably something like this:
```
if create_sg_rolemapping.ok():
   return
```